### PR TITLE
Add a Zendesk dlt source example

### PR DESCRIPTION
**Pull request by burnash**

### CODE REVIEW

File: zendesk_to_weaviate.ipynb [hunk 1]
```diff
@@ -0,0 +1,557 @@
...
+        "        api_key='YOUR_WEAVIATE_API_KEY'\n",
+        "    ),\n",
+        ")\n",
+        "response = (\n",
+        "      client.query\n",
+        "      .get(\"Tickets\", [\"subject\", \"description\", \"created_at\"])\n",
+        "      .do()\n",
+        ")\n",
+        "\n",
+        "response_to_table(response, 'Tickets', [\"subject\", \"description\", \"created_at\"])"
```
Comment (AstrakhantsevaAA) [inline zendesk_to_weaviate.ipynb:180]: I can't find response_to_table function, it's confusing

Comment (burnash) [reply zendesk_to_weaviate.ipynb:180]: Good point, I think I've lost it along the way.

Comment (burnash) [reply zendesk_to_weaviate.ipynb:173]: Hmm, I have it in the colab version: https://colab.research.google.com/drive/1nxaIOUtuW7mdHcHZlHQyn578bvSN2pq. Might be an export issue.

**Comment by burnash** [comment]
@AstrakhantsevaAA added response_to_table()

**Pull request merged**